gta/dff_importer.py

```diff
@@ -511,9 +515,10 @@ def remove_object_doubles():
             bmesh.ops.remove_doubles(bm, verts = bm.verts, dist = 0.00001)
 
             # Add an edge split modifier
-            object   = self.objects[frame]
-            modifier = object.modifiers.new("EdgeSplit", 'EDGE_SPLIT')
-            modifier.use_edge_angle = False
+            if frame in self.objects:
```

Comment:
And here I got frame of -1, and also index out of range. Once I took care of these, everything imported smoothly.